### DISCUSSION TITLE
How to remove failed tilt series permanently?

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#51] [General]
A couple of tilt series failed during preprocessing.  Now that I am going back to that block to pick particles, nextPYP tries to process these already failed tilt series again and again.  How do I remove these permanently?

**Comment by abartesaghi** [comment]
How are the tilt-series failing? Do you have any error messages to share? Right now, there is no way to remove a subset of files from a project through the web interface.

**Comment by JuhaHuiskonen** [comment]
I think they each had a corrupted tilt image, the error was about TIF file could not be read or similar.
